src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let rental = 0;
+  const d = days;
+
+  rental = d * 40;
```

Comment:
```suggestion
  let rental = days * 40;
```
no need in `d` variable